readme.md

```diff
@@ -43,7 +43,7 @@ Where we're picking up the app, it has only a functioning index route that uses
 
```

Comment:
^ I think where we're picking up the app, it has a functioning show route too....